docs/quadratic/Interact with nft.md

```diff
@@ -0,0 +1,95 @@
+
+# Deploy the NFT Contract
```

Comment:
This looks wonderful. If it fits within your comfort zone, we might explore using cosmjs for deployment and related activities, given that the Weco team plans to use cosmjs for their integration efforts.